zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/repo/ExporterRepository.java

```diff
@@ -90,7 +91,8 @@ private void validate(final ExporterDescriptor descriptor) throws ExporterLoadEx
     try {
       final Exporter instance = descriptor.newInstance();
       final ExporterContext context =
-          new ExporterContext(LOG, descriptor.getConfiguration(), NULL_PARTITION_ID);
+          new ExporterContext(
+              LOG, descriptor.getConfiguration(), NULL_PARTITION_ID, new SimpleMeterRegistry());
```

Comment:
:thought_balloon: Remember to update the documentation [here](https://docs.camunda.io/docs/self-managed/concepts/exporters/#loading) after to add a note that the meter registry you get during the loading/validation phase is just an in-memory one, and will be discarded after.

Reply:
Does the user need to know this? My understanding is if they add code in the `configure` block registering meters it will add it to the simple register, discard it, and then the second time round register it with the actual `PrometheusMeterRegistry` as intended.

Reply:
https://github.com/camunda/camunda-docs/pull/4082 documentation changes are here